### PR TITLE
split the conference list yearwise

### LINKED ISSUE #28: Make the list yearwise
The page is getting longer day to day.
How about listing the conferences by year
For ex: We mention the year in the readme and a link which redirects to the conferences of that year.

### PR DESCRIPTION
**Pull request by shivakrishna9**
Fix #28

### CODE REVIEW

File: 2016/2016.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+## 2016
+
+| Conference | Date | Venue | Description |
+|------------|------|-------|-------------|
+|[Mini Deb-Con](http://in2015.mini.debconf.org/)|30 - 31 January| Andheri (E), Mumbai | It is an awesome opportunity for any user who wants to experience the magic of FOSS |
```
Comment (nikhita) [inline 2016/2016.md]: The list order is no more chronological now. It's reverse chronological (see https://github.com/nikhita/tech-conferences-india/pull/29).

Can you update the list to how it occurs in the current README now?

File: 2018/2018.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+## 2018
+
+| Conference | Date | Venue | Description |
+|------------|------|-------|-------------|
+| [GopherCon India](http://www.gophercon.in/) | 09-10 Mar | Sheraton Grand, Sangamvadi, Pune | Go conference in India. Go is an open source project developed by a team at Google and many contributors from the open source community. |
```
Comment (nikhita) [inline 2018/2018.md]: There are more conferences listed out for 2018 in the README. Please add all of them.

File: README.md [hunk 3]
```diff
@@ -6,15 +6,6 @@ To add a conference to the list, please [contribute](#contribute)!
 
 If you would like to be notified of future conferences that would be added to the list, you could watch this repo. New additions are made via PRs and have full description of the conference so that you can view it directly in your email. ([Example PR](https://github.com/nikhita/tech-conferences-india/pull/25))
 
-## 2018
-
-| Conference | Date | Venue | Description |
-|------------|------|-------|-------------|
-| [Great Indian Developer Summit](http://www.developermarch.com/developersummit/index.html) | 24-27 Apr | Bengaluru | 	With over 40000 attendees benefiting from ten game changing editions, GIDS is the gold standard for India's software professional ecosystem.
```
Comment (nikhita) [inline README.md]: All these conferences are missing in the current `2018.md` file.

**Comment by nikhita** [comment]
Also, can you squash the commits to a single commit? I can see a merge commit present now.

**Comment by shivakrishna9** [comment]
@nikhita this occured because I didnot pull the updated repo. Will close this and give another PR again.

**Comment by nikhita** [comment]
@shivakrishna9 can you update in this PR only? You can `git reset` or `git rebase`. It's better to continue in the same PR to not loose context. :)

But if you are having trouble with continuing in this PR, then another PR is fine.

**Pull request closed without merging**